app/src/main/java/com/group80/uoftinder/CreateAccountView.java

```diff
@@ -0,0 +1,264 @@
+                String userAge = ((EditText) findViewById(R.id.age)).getText().toString();
+
+                ChipGroup identity_group = findViewById(R.id.identityGroup);
+                int identity_count = identity_group.getChildCount();
+                String identity = "";
+
+                //loops through all chips(answers) of identity answers and finds which chip was
+                //selected
+                for(int i = 0; i<identity_count; i++) {
+                    Chip chip  = (Chip) identity_group.getChildAt(i);
+                    if(chip.isChecked()) {
+                        identity = chip.getText().toString();
+                    }
+                }
+                //loops through all chips(answers) of type answers and finds which chip was
+                //selected
+                ChipGroup type_group = findViewById(R.id.typeGroup);
+                int type_count = type_group.getChildCount();
+                String type = "";
+                for(int i = 0; i<type_count; i++) {
+                    Chip chip  = (Chip) identity_group.getChildAt(i);
+                    if(chip.isChecked()) {
+                        type = chip.getText().toString();
+                    }
+                }
+
+                //checks if all information was entered correctly
+                boolean move_on = control.newAccount(userName, userAge, identity, type);
+                if(move_on) {
+                    //sets information for the currentUser
+                    currentUser.setName(userName);
+                    currentUser.setAge(Integer.parseInt(userAge));
+                    currentUser.setGender(identity);
+                    currentUser.setUserType(type);
+
+                    createQuestionnaireView(currentUser, "Academic");
+                }
+                else {
+                    String text = "Please enter your information correctly";
+                    TextView error = findViewById(R.id.error_basicinfo);
+                    error.setText(text);
+                }
+
+            }
+        });
+
+    }
+
+    /*
+     * Creates questionnaire view for the correct type
+     * @param type      a string representing the type of user that the user selected (only
+     *                  "academic" for now)
+     */
+    private void createQuestionnaireView(User currentUser, String type) {
+        if(type.compareTo("Academic")==0) {
+            createAcademicQuestionnaire(currentUser);
+        }
+//        else if(type.compareTo("Friendship")==0) {
+////           friendship questionnaire view
+//        }
+//        else if(type.compareTo("Romantic")==0) {
+////            academic questionnaire view
+//        }
+    }
+
+    /*
+     * Creates the academic_questionnaire.xml and gets inputs, proceeds to recommendation page if
+     * information is entered correctly
+     */
+    private void createAcademicQuestionnaire(User currentUser) {
```

Comment:
Similar to [before](https://github.com/CSC207-2022F-UofT/course-project-group-80/pull/24/files#r1027249181), maybe move the following lines into a new view, then use `Intent` to switch to that view?